launch/start.launch

```diff
@@ -24,6 +24,7 @@
     <arg name="robot_name" default="hsrb" />
     <arg name="personal_name" default="hero" />
     <arg name="enable_emergency_stop" default="false" />
+    <arg name="gui" default="false" />  <!-- Gazebo GUI -->
```

Comment:
Change to `gazebo_gui` and add comment it is not forwarded to the hardware manager.

Reply:
done